### PR TITLE
(FAVOR FOR ARATHIAN) Edited Gygax diamond cost to be silver and gold but the same values

### PR DESCRIPTION
**Pull request by MacBlaze1**
## About The Pull Request

Gygax diamond values changed to their equivalents in silver and gold, therefore if a part costs 2k diamonds, it will now cost 2k gold and 2k silver. This PR is a favor for Arathian Spider RD main man on Discord and the servers who wanted to change the costs but didn't know how to make multiple 1 lines fixes. Seeing as I can make just over 1 line fixes, I did this as a favor.

## Why It's Good For The Game
Gygax has been heavily nerfed due to autopunch removal? No longer sleeps target and does less damage?? ((Im not a fucking science main) This brings the Gygax cost for parts into line with its abilities, as it is no longer a fucking death mobile running around chain stunning and unlocking people to death, it is now a weaker Durand. This PR hopes to make it so it is not so outrageously priced for what you get.

## Changelog
:cl:
balance: Rebalanced Gygax part diamond cost numbers to be the same but now in gold AND silver.
/:cl:

### CODE REVIEW

**Comment by Dawson1917** [comment]
>heavily nerfed

still a good mech and easy to bone in

>no longer a fucking death mobile

yes it is

**Comment by MacBlaze1** [comment]
> > heavily nerfed
> 
> still a good mech and easy to bone in
> 
> > no longer a fucking death mobile
> 
> yes it is
@Dawson1917 I just made this for the spider man, okay? Hes the science main.

**Comment by Arathianen** [comment]
Hi, yes, I kept bitching about this.

Gygax actuator isn't actually entirely removed. Instead its autopunch was entirely removed. Its punch was also specifically nerfed to do less damage AND not sleep the target.

Diamonds, for anyone who does science, are by far the most limited resources with multiple uses for many departments.

Gygax was previously the most powerful mech due to its autopunch ability along with its sleep. Right now it is the weakest combat mech (yes, weaker than the HONK too).

This brings its cost in line with its abilities to be the budget mech that isn't very powerful but is fast and can be used in a pinch over the durand.

**Comment by Arathianen** [comment]
One more thing, since already the "LOL BUFFING MECHS EVER" brigade is in:

Gygax right now is removed from the game for anyone who knows how the meta. Durand is the better combat mech and phazon is the better faster mech which costs less than the gygax if you get your hands on an anomaly core.

I think an argument can be made for nerfing the gygax's stats a bit more to go with the cost changes, but to say the gygax is fine right now is stupid.

**Comment by Dawson1917** [comment]
arathianen im sorry if you died in a gygax but they are objectively a piss easy way to multiply your murdering capacity in every single round as a roboticist or rd

if miners are bringing in enough mats to build _any_ combat mechs then there'll also be enough diamond to make a gygax/phazon. 
if you think it's not enough to have stun immunity, damage slowdown immunity, a multiplied health bar, and stronger guns than you'll find in the armory, all being available to you every single round, then that is an issue for you. nobody else. mechs, gygax included, are still far too strong and mass-produceable in their current state to warrant being made any easier to construct.

**Comment by SkeletalElite** [comment]
> arathianen im sorry if you died in a gygax but they are objectively a piss easy way to multiply your murdering capacity in every single round as a roboticist or rd
> 
> if miners are bringing in enough mats to build _any_ combat mechs then there'll also be enough diamond to make a gygax/phazon.
> if you think it's not enough to have stun immunity, damage slowdown immunity, a multiplied health bar, and stronger guns than you'll find in the armory, all being available to you every single round, then that is an issue for you. nobody else. mechs, gygax included, are still far too strong and mass-produceable in their current state to warrant being made any easier to construct.

Literally all of those things apply to the durand and the gygax is more expensive than the durand because ???

**Comment by Dawson1917** [comment]
> Literally all of those things apply to the durand and the gygax is more expensive than the durand because ???

because the durand's cost should be increased, instead of decreasing the gygax's? they are both laughably easy to make

**Comment by Arathianen** [comment]
I am talking to the guy on discord and his argument basically is "mechs are bad and shouldn't exist in the game".

If somebody thinks that, ok, but that is irrelevant to the fact that we have mechs and they have a certain power curve and the gygax is way below said power curve.

Basically, dawson's opposition to this PR is not in good faith.

**Comment by Arathianen** [comment]
I am talking to the guy on discord and his argument basically is "mechs are bad and shouldn't exist in the game".

If somebody thinks that, ok, but that is irrelevant to the fact that we have mechs and they have a certain power curve and the gygax is way below said power curve.

Basically, dawson's opposition to this PR is not in good faith.

**Comment by Dawson1917** [comment]
>Basically, dawson's opposition to this PR is not in good faith.

image

this is mech powercreep by making the easy murderbone machines even easier to make. if you think durands cost too little compared to the gygax then increase the durand's cost.

**Comment by necromanceranne** [comment]
Good as heck for the price, so seems fine to me. Mobility is a powerful tool but Durand's would really do so much more than what the Gygax could in a prolonged bone. Gygaxes die pretty quickly to concentrated fire in enclosed spaces.

Also, given the node position, it is entirely possible for this mech to appear in war ops now!

**Comment by Arathianen** [comment]
Yeah, the idea is for the gygax to be the budget mech that you can make before any other mech :)

That is why I said I think a good argument could be made for further nerfing its stats, but that should come after letting the new price and meta settle down a bit. If people spam gygaxes, it should be nerfed to be the real "quick, we need a mech cause a thing is happening" mech.

**Comment by Swiftkor** [comment]
Kind of off topic but I wish there was a lower tier combat mech that didn’t have access to all the crazy lethal stuff that the other mechs have. Kind of a security non-lethal focused mech that could be made relatively earlier in the round.

**Comment by Whoneedspacee** [comment]
durand can literally use all the guns too.

gygax pretty much offers nothing new to the table anymore except speed which is basically useless since it drains your power to shit (and really isn’t better than just having an extra gun or more defense)

all for the price of a stupid amount of diamonds people steal for other shit

i don’t support this change though, the gygax should be changed to provide a unique mech thing, maybe capitalize on the speed thing.

don’t bring back the autopunching though that basically auto destroys any stationary targets and mega stun to death combo.

literally run through a blob.

**Comment by necromanceranne** [comment]
I argued the same thing on tlal's PR but he was just looking to kneecap the mech because he probably died to it and didn't really give a shit about the state of the mech after his change.

It's quite garbage but it was always going to be like that given it was held up by the flimsy string that was autopunches.

**Comment by Arathianen** [comment]
@Swiftkor other servers have "locker mechs" which are basically ghetto maint mechs. That would be a very cool feature.

@Whoneedspacee this is just a quick rebalance PR. If you have ideas and wanna do a full redo of the mech, I would absolutely support that. This PR is just to have the gygax not be dogshit for the time being. I agree gygax's ability, after the nerf, is fairly boring now that the autopunch component is gone.

**Comment by kriskog** [comment]
Having the extra speed from the leg actuators makes the gygax very useful, and it can still fit guns. I fail to understand how you think the removal of the autoattacks and hard stun is enough to make this mech worthless. It's a bit less bullshit than before, but it is still quite strong due to its ability to retreat.

Silver and gold are the most underused and abundant "rare" materials there is. Making a "light mech" from fairly heavy metals is also a weird choice. At the very least it should be brought in line with the durand, since it fits within the same power level. I am still of the opinion it should cost more though.

**Comment by CRITAWAKETS** [comment]
The Gygax right now is currently just useless. Anybody saying that the Gygax right now is too durable is very new or they straight up are just trolling.

Why is that? Integrity of 250, 25 melee armor. 312.5 effective melee health. For just 27 toolbox hits, you can get rid of a gygax. Mechs also take gradual damage to systems and such. Pray to Saradomin that they dont have a fire axe or chainsaw because it only takes 15 hits for the chainsaw or 14 for the fire axe. Its surprisingly easy to get that many hits in and it really requires them to use cheese tactics (scattershot is the strongest non-admin weapon in the game tbh) to not die as fast.

**Comment by kriskog** [comment]
If you are taking 27 toolbox hits in a gygax, you aren't using it right.

**Comment by Arathianen** [comment]
> If you are taking 27 toolbox hits in a gygax, you aren't using it right.

The point is about how weak the gygax is.

Know what DOES destroy the gygax in one volley? An Ion gun. Know what ion gun does? It completely disables the weapons of the gygax.

"It moves a bit faster" is laughable. Phazons move at the same speed and can phaze through walls and they cost less than a gygax and even phazons aren't widely considered very good.

**Comment by kriskog** [comment]
The phazon requires an anomaly core, those aren't all that easy to get. Sure as heck a lot harder to get than some silver and gold. 

Ion guns are intended to be a hard counter to mechs and borgs, I'm not sure why you bring it up as some kinda argument that the mech should be cheaper.

**Comment by Ratvar** [comment]
You don't need to be a science main to be a powergamer buffing OP stuff under a disguise of a promise for a science main. Please, help "servers who wanted to change the costs" by going to their github.
This doesn't " brings the Gygax cost for parts into line with its abilities" because all mechs are "fucking death mobiles running around ", all at cost of miners existing at all. That roboticists are too braindamaged to consider using them in a smart way that doesn't result in trying to melee assistants is another issue entirely.
Nerfs do not require corresponding buffs.

**Comment by MacBlaze1** [comment]
> You don't need to be a science main to be a powergamer buffing OP stuff under a disguise of a promise for a science main.


:omegalul: I havent played scientist in months, maybe even a year. Why dont you get off your ass and go check the #coding-general discord logs and find out how I am a science shill :tm: and want to ruin tg?

**Comment by Dawson1917** [comment]
> The Gygax right now is currently just useless. Anybody saying that the Gygax right now is too durable is very new or they straight up are just trolling.
> 
> Why is that? Integrity of 250, 25 melee armor. 312.5 effective melee health. For just 27 toolbox hits, you can get rid of a gygax.

Stop standing still while an assistant tickles you to death.
Having an extra suit of 300 health on top of your own health (humans can take 100 damage before falling into crit, not even counting how much armor you're wearing inside your mech), complete stun immunity, weapons capable of critting in 1-3 hits, no damage slowdown, and fireaxe-damage fists, aren't all enough for you? You aren't satisfied in getting mechs in 95% of rounds without having to do work? You want MORE? 

Science players are actual infants.

**Comment by Whoneedspacee** [comment]
durand was still the 2nd best mech before the gygax nerf due to these things, and some even preferred it to the gygax. so it makes sense now that the gygax doesn’t have punchspam it isn’t good at all.

speed was never a utility of the gygax. top speed is 8/10 times a players run speed. you might as well just get a stun on them with the durand.

ranged weapons massively fuck you if you’re in a gygax which was previously stated as well.

just go fight nuke ops with a durand and with a gygax and tell me which does better, 100% it’s the durand.

gygax needs to move faster at a cheaper cost, the only reason it cost so much was because the autopunch was op. then maybe you can justify the ludicrous diamond and research costs.

**Comment by Whoneedspacee** [comment]
@Dawson1917 literally all you’ve stated are things that the durand gets, but better because it’s cheaper, and it’s also just tougher integrity and armor wise.

this is the entire issue, the gygax is so much worse now.

**Comment by CRITAWAKETS** [comment]
Dawson, not only does it take so much damage from a fire axe that anybody can reasonably get rid of a durand, but also, you kind of forgot the part where mechs take systems damage. I would say its more like 4 hits before your mech decides "OY BLYAT MY SHITS BROKE"

**Comment by Dawson1917** [comment]
you are literally complaining that it is possible to lose in your free stunproof suit of armor weapons platform if people grab the strongest weapons on the station and you're too incompetent to hit them once

**Comment by CRITAWAKETS** [comment]
> you are literally complaining that it is possible to lose in your free stunproof suit of armor weapons platform

> free stunproof suit of armor weapons platform

A gygax ISNT FREE
A gygax HAS A STUN MECHANIC, A REALLY EASY ONE TO USE TOO

how 2 brek mekk for dummy

you can use your local chainsaw and just go where they arent facing, but that has a tiny amount of danger, so here is what you do

1) get uranium
2) get metal
3) grind both of them in seperate beakers
4) mix near mech
5) toolbox or whatever weapon is your favorite agaisnt the mech
6) watch as its both stunned and also has its systems fucked to hell and back

**Comment by AcapitalA** [comment]
Balance aside, I'd say this is a good change because it stops roboticists from draining the entire diamond supply while it also brings gold into mech construction.

**Comment by FloranOtten** [comment]
Good change. People who aren't interested in mech balance and are just here to try and slowly get mechs removed BTFO

**Comment by ExcessiveUseOfCobblestone** [comment]
Why is the baseline of combat utility the mech specifically designed to tank and fight?

The baseline at which these things are compared shouldn't even be a mech, it should be your average crewman.

That's not to say the gygax is or isn't as powerful as the durand, but it's certainly powerful in it's own right. I could really give a toss about it's comparability to the durand if it's still a really strong tool which people can easily cull the station with.

**Comment by Arathianen** [comment]
@cobby:

I am confused about your comment.

The gygax obviously is strong. If you own a gygax, you own an advantage.

However, when you are at the mech lathe and decide what to print, you have the option between 3 mechs. A proper (good) balance would be so that all 3 mechs can be considered comparable to each other with their own unique strengths and differences. The gygax, on the power curve AND the tech tier is lower than the other 2 mechs.

It is also, by far, the most expensive mech. Since the nerf is live, I have seen 0 gygaxes being created due to that.

To put it another way. Imagine a c2 chemical that healed brute and did some lung damage or whatever. It was balanced to compared to other chems. Bit shittier than some, but not the shittiest.

Now imagine if it was the hardest chem to make by far in the game.

No one would make it, not because the chemical is inherently shit but because the resource and time expenditure wouldn't correspond to the reward.

That is the situation with the gygax. The gygax, standalone, is strong but what it costs to get it is absurdly high.

**Comment by Arathianen** [comment]
Also, furthermore:

>if it's still a really strong tool which people can easily cull the station with

It's no longer that. The nerfs were **devastating** to the gygax. The gygax right now dies easily, has many direct counters and no particularly big strength. If someone can murderbone a whole station with a gygax, something is seriously wrong.

**Comment by ExcessiveUseOfCobblestone** [comment]
@zxaber

**Comment by Arathianen** [comment]
Zxaber has upvoted this PR, but I would love his more detailed thoughts too 😃

**Comment by zxaber** [comment]
The Gygax autopunching was pure cancer, but _did_ constitute a large part of the Gygax's damage potential and power. Having just over 60% the health of a Durand (and only 50 above the Ripleys), lower armor in most categories, and a lower tolerance for heat, the Gygax is not a particularly tanky mech. While faster speed is a nice feature to have when trying to respond to threats or run away from something, it doesn't always help once you're actually in the fight. Autopunching having the ability to delete walls (including R-walls within a few punches at most) and just run people over was the main reason for the mech being worth the diamonds.

It would be cool if someone could come up with a good replacement for autopunching; something that gives the Gygax meaning and stays within the theme of speed without feeling so bullshit. In the absence of that, this price adjustment is a good alternative. (I'm not sure how tacking on a heavy metal like gold helps a mech be lightweight, but that's not a balance point).
